components/dashboard/src/teams/JoinTeam.tsx

```diff
@@ -24,16 +24,15 @@ export default function JoinTeamPage() {
                     throw new Error("This invite URL is incorrect.");
                 }
                 const team = publicApiTeamToProtocol((await teamsService.joinTeam({ invitationId: inviteId })).team!);
```

Comment:
More of a comment on how this can evolve in the future. If we put the joinTeam call into a org-related mutation, it helps centralize how we can make sure that invalidates the related org querie(s). Not saying we should do that as part of this work though, just a callout.

Reply:
I intentionally opted for handing out a generic invalidate function to those client that mutate server state, because there was always only one caller and we couldn't really benefit from other callbakcs that a mutation would provide. But, yes, definitely makes sense if we have mutations that we want to optimize with e.g. optimistic changes.